### ISSUE TITLE
2.1 PDF Printing

### ISSUE THREAD

**Issue by klabarge** [qzind/tray#91]
When I try Print PDF from the `sample.html` page I get the below error.

Note that I tested the sample from both a webserver and the file system.

```js
sample.html:1890 Error: Cannot parse (FILE)assets/pdf_sample.pdf as a PDF file(…)
```

**Stack trace:**
```
[ERROR] 2016-08-04 18:34:29,879 @ qz.utils.PrintingUtilities:?
	Failed to print
java.lang.UnsupportedOperationException: Cannot parse (FILE)assets/pdf_sample.pdf as a PDF file
	at qz.printer.action.PrintPDF.parseData(Unknown Source)
	at qz.utils.PrintingUtilities.processPrintRequest(Unknown Source)
	at qz.ws.PrintSocketClient.processMessage(Unknown Source)
	at qz.ws.PrintSocketClient.onMessage(Unknown Source)
	at sun.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.jetty.websocket.common.events.annotated.CallableMethod.call(CallableMethod.java:70)
	at org.eclipse.jetty.websocket.common.events.annotated.OptionalSessionCallableMethod.call(OptionalSessionCallableMethod.java:68)
	at org.eclipse.jetty.websocket.common.events.JettyAnnotatedEventDriver$2.run(JettyAnnotatedEventDriver.java:210)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.net.MalformedURLException: no protocol: assets/pdf_sample.pdf
	at java.net.URL.<init>(Unknown Source)
	at java.net.URL.<init>(Unknown Source)
	at java.net.URL.<init>(Unknown Source)
	... 13 more
[TRACE] 2016-08-04 18:34:29,897 @ qz.utils.PrintingUtilities:?
	Returning processor back to pool
```

Per the documentation, the `format` and `flavor` should default to the proper values.  When I change the `options` to:

```js
 var printData = [
            { type: 'pixel', data: 'assets/pdf_sample.pdf' }
        ];
```
I receive a `Error: NullPointerException` but an identical stack trace.

**Comment by bberenz** [comment]
Looks like my changes to qz-tray.js went missing after one of the rebases and I didn't catch it.
Should all be fixed in dc97eccec6d258b6742a78f8d1792595bd3f4b73

**Comment by klabarge** [comment]
> Should all be fixed in dc97ecc

Tested and works :+1:
